src/stylus/components/_sidebar.styl

```diff
@@ -7,7 +7,7 @@
   width: $sidebar-width
   max-width: 100%
   z-index: 3
-  @extend .z-depth-2
+  elevation(16)
```

Comment:
Is elevetion 16 based on material-specs? I think its little bit too much. I think elevation(8) if is more suitable. I use a 4K monitor and it just looks bit off.